src/app/js/public/Export.js

```diff
@@ -1,41 +1,84 @@
-import React, { PropTypes } from 'react';
+import React, { Component, PropTypes } from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import compose from 'recompose/compose';
+import translate from 'redux-polyglot/translate';
 import { List } from 'material-ui/List';
+import Subheader from 'material-ui/Subheader';
 
-import { exportPublishedDataset as exportPublishedDatasetAction } from '../public/export';
-import config from '../../../../config.json';
+import { fromExport } from './selectors';
+import {
+    loadExporters as loadExportersAction,
+    exportPublishedDataset as exportPublishedDatasetAction,
+} from '../public/export';
 import ExportItem from './ExportItem';
+import { polyglot as polyglotPropTypes } from '../propTypes';
 
-export const ExportComponent = ({ handleExportClick, uri }) => (
-    <List className="export">
-        {
-            config.exporters.map(type => (
-                <ExportItem
-                    key={type}
-                    type={type}
-                    uri={uri}
-                    onClick={handleExportClick}
-                />
-            ))
+export class ExportComponent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { exportedFields: [] };
+    }
+
+    componentDidMount() {
+        this.props.loadExporters();
+    }
+
+    handleSelectedFieldsChange = (exportedFields) => {
```

Comment:
Because I need the `componentDidMount`. if I have to make a component, I don't need recompose